cassdegrees/static/js/staff/globalrequirements.js

```diff
@@ -83,22 +83,23 @@ Vue.component('global_requirement_general', {
         const request = new XMLHttpRequest();
 
         request.addEventListener("load", () => {
-            rule.check_options();
+            rule.check_options(false);
         });
         request.open("GET", "/api/search/?select=code&from=course");
         request.send();
     },
     methods: {
-        check_options() {
+        check_options(is_submission) {
             this.invalid_units = this.details.unit_count <= 0;
             this.invalid_units_step = this.details.unit_count % 6 !== 0;
             this.units_is_blank = this.details.unit_count === "";
 
-            this.is_invalid = !this.details.courses1000Level && !this.details.courses2000Level && !this.details.courses3000Level
-                && !this.details.courses4000Level && !this.details.courses5000Level && !this.details.courses6000Level
-                && !this.details.courses7000Level && !this.details.courses8000Level && !this.details.courses9000Level
-                && this.details.subject_area === "";
-
+            if (is_submission) {
+                this.is_invalid = !this.details.courses1000Level && !this.details.courses2000Level && !this.details.courses3000Level
+                    && !this.details.courses4000Level && !this.details.courses5000Level && !this.details.courses6000Level
+                    && !this.details.courses7000Level && !this.details.courses8000Level && !this.details.courses9000Level
+                    && this.details.subject_area === "";
+            }
```

Comment:
This only changes `is_invalid` when the `is_submission` flag is set

Reply:
intentional, the message that shows up instatntly depends on is_invalid, hence i added the is_submission condition

Reply:
Still think this needs to be addressed. If they correct their error the error message still appears. This functionality could confuse the user